web/src/components/Create.js

```diff
@@ -41,6 +41,7 @@ class Create extends Component {
                         <input defaultChecked type="radio" id="both" name="view" onChange={handleChange} value="both" /> 
                     </div>
                     </form>
+                    {/* CR * routes instead */}
```

Comment:
If you are showing a component based on a single state property like this, and that component is meant to be a section, you MIGHT have an opportunity to do this with React Router instead! I don't know if it is truly the best approach here but something to keep in mind.